src/dispatch/case/flows.py

```diff
@@ -23,12 +24,48 @@
 from dispatch.ticket import flows as ticket_flows
 
 from .models import Case, CaseStatus
-from .service import get, delete
-
+from .service import delete, get
 
 log = logging.getLogger(__name__)
 
 
+def create_conversation(case: Case, db_session: SessionLocal):
```

Comment:
We should move `create_conversation` and `update_conversation` to a new conversation.flows module at some point. Perhaps when we refactor the incident create flow.